BondageClub/Scripts/Typedef.d.ts

```diff
@@ -548,7 +551,10 @@ interface Character {
 	GetDeafLevel: () => number;
 	IsLoverPrivate: () => boolean;
 	IsEdged: () => boolean;
+	IsPlayer: () => boolean;
```

Comment:
Little improvement, that will allow even nicer typings with TS
```suggestion
	IsPlayer: () => this is PlayerCharacter;
```

Reply:
Oh, good call.